woocommerce/woocommerce-android
# Hackweek: Fix toolbar color

### Description

As discussed here p1677743676493199-slack-C6H8C3G23 we want to bring consistency across toolbar colors on different screens in the dark mode. Currently, on compose screens toolbar color is `woo_black_90`, while on XML screen it's `woo_black_900`. On Figma we have `woo_black_90` with an overlay which is added if the toolbar has a shadow. This PR attempts to align with Figma approach

### Testing instructions



Start in both dark and light modes:
* Main
* Settings
* Support

screens and notice that there no visual changes

---


Start Chrome Tab, for instance, from the inbox screen - notice that toolbar is grayer now and matches other places

### Images/gif

<img width="483" alt="image" src="https://user-images.githubusercontent.com/4923871/223092738-08650270-495e-41e2-aa5d-a9bcde65ba98.png">
